api/proto/teleport/legacy/types/types.proto

```diff
@@ -5621,11 +5621,33 @@ message PluginOktaSettings {
   // OrgUrl is the Okta organization URL to use for API communication.
   string org_url = 1;
 
-  // EnableUserSync controls the user sync in the Okta integration service.
+  // EnableUserSync controls the user sync in the Okta integration service. Deprecated.
+  // TODO(mdwn): Remove once e changes have been made.
   bool enable_user_sync = 2;
```

Comment:
Can we use `deprecated = true` proto option on the fields you moved inside the new message? Or are you planning to just remove them (add add "reserved" fields) after "e" changes land?

Reply:
I'm planning on the latter, actually.